eventio/event_usage.go

```diff
@@ -17,6 +17,7 @@ type UsageEvent struct {
 	SpaceGUID     string `json:"space_guid"`
 	SpaceName     string `json:"space_name"`
 	PlanGUID      string `json:"plan_guid"`
+	PlanUniqueID  string `json:"plan_unique_id"`
```

Comment:
These names (`planGUID` and `PlanUniqueID`) are really confusing because they are both unique and they are both GUIDs.

Reply:
It is also confusing for me, but naming is hard. `guid` here refers to the fact that is a GUID https://betterexplained.com/articles/the-quick-guide-to-guids/. All the columns all around have that `_guid` suffix. 

But then we have two kind of `plan_guid`:
 - `plan_guid` as the GUID of the metadata reported in the events. This GUID is different across deployments, and changes if the broker is deregistered and registered again.
 -  `unique_id` reported by the CF api: https://apidocs.cloudfoundry.org/4.0.0/service_plans/list_all_service_plans.html and which is feed into by the broker. This is constant across installations.

Reply:
How about we call it something like `plan_catalog_guid` to indicate that it comes from the service catalog returned by the broker. Or is that more confusing given the various layers of indirection here?